core/src/config.rs

```diff
@@ -53,12 +55,29 @@ pub struct Admin {
     pub public_key: VerifyingKey,
 }
 
+#[cfg_attr(test, derive(Clone, PartialEq))]
 #[derive(Debug, Deserialize, Serialize)]
 pub struct Settings {
     pub admin: Admin,
 }
 
-#[derive(Debug, Deserialize, Serialize)]
+/// Returns the default derivation path per config version.
+pub const fn default_derivation_path(config: ConfigDiscriminants) -> u32 {
+    match config {
+        // TODO: what should these be?
+        ConfigDiscriminants::V1 => 2,
+        ConfigDiscriminants::V2 => 2,
+        ConfigDiscriminants::V3 => 2,
```

Comment:
i _really_ don't know what we need or want these to be

@JettTech @mattgeddes 

any insight?